core/app/models/spree/store_credit_category.rb

```diff
@@ -1,6 +1,6 @@
 class Spree::StoreCreditCategory < Spree::Base
-  class_attribute :non_expiring_credit_types
-  self.non_expiring_credit_types = [Spree.t("store_credit.non_expiring")]
+  class_attribute :non_expiring_credit_categories
```

Comment:
We were just looking at the Store Credits functionality this morning and agree with @luukveenis's assessment. We found the inconsistency in naming pretty confusing (note it's also inconsistent in the backend UI).

Also, we're actually lost as to what Category and Types are trying to achieve? I.e. what is the difference? Wondering if anyone here could shed some light onto this. From what we could see it didn't seem like there needed to be two models representing this concept.